src/index.js

```diff
@@ -3,6 +3,13 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+(function setPath() {
+  process.env.PATH = [
+    `${process.cwd()}/node_modules/.bin/`, process.env.PATH 
+  ].join(path.delimiter);
+})();
 
```

Comment:
Yep, I was just looking at that. One option here would be to set the path at the start of run and then reset it when run is complete. That's not necessarily super easy though being as run could be async.